hoot-core-test/src/test/cpp/hoot/core/TestUtils.h

```diff
@@ -219,15 +219,27 @@ class HootTestFixture : public CppUnit::TestFixture
     ResetAll
   };
 
-  /** Constructor to set the default reset to none */
-  HootTestFixture() : _reset(ResetNone) { }
+  /** Constructor to set the paths, default reset to none, and create the output path if needed */
+  HootTestFixture(const QString& inputPath = UNUSED_PATH, const QString& outputPath = UNUSED_PATH)
+    : _inputPath(inputPath),
+      _outputPath(outputPath),
+      _reset(ResetNone)
+  {
+    if (outputPath != UNUSED_PATH)
+      TestUtils::mkpath(_outputPath);
+  }
```

Comment:
Added the creation of the `_outputPath` here in the base class that creates the output directory if needed.